pkgs/applications/misc/hunter/default.nix

```diff
@@ -0,0 +1,66 @@
+{ stdenv
+, callPackage
+, makeRustPlatform
+, fetchFromGitHub
+, IOKit ? null
```

Comment:
Please use inherit in top-level to not default this to null cause this will always fail on darwin.